autogpt_platform/backend/backend/blocks/basic.py

```diff
@@ -41,8 +41,7 @@ class Output(BlockSchema):
     def __init__(self):
         super().__init__(
             id="1ff065e9-88e8-4358-9d82-8dc91f622ba9",
-            description="This block forwards the `input` pin to `output` pin. "
-            "This block output will be static, the output can be consumed many times.",
+            description="This block forwards an input value as output, allowing reuse without change.",
```

Comment:
The static information is really important but Idk what is the better way to convey it.
This is a special block when you use it, the link connected to other nodes will look and behave differently:

<img width="1134" alt="image" src="https://github.com/user-attachments/assets/351b2670-6579-4de3-bcf8-7273dfa23079">

The output produced by this block can be reused and not gone when its consumed.